torch/csrc/aot_inductor/c/utils.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <assert.h>
+
+#ifdef __GNUC__
+#define AOT_INDUCTOR_EXPORT \
+  __attribute__((__visibility__("default"))) __attribute__((used))
+#else // !__GNUC__
+#ifdef _WIN32
+#define AOT_INDUCTOR_EXPORT __declspec(dllexport)
+#else // !_WIN32
+#define AOT_INDUCTOR_EXPORT
+#endif // _WIN32
+#endif // __GNUC__
+
+#define AOT_INDUCTOR_CHECK(cond, msg) \
+  { assert((cond) && (msg)); }
```

Comment:
Still not sure why this needs to be in the public headers. Also still not sure about the use of C assert construct

Reply:
In particular, assert will be compiled unless you do a debug build (which we don't by default)

Reply:
Yeah, without `NDEBUG` all asserts will be wiped out by the preprocessor